submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,106 @@
+
+According to Steam, I’ve played almost 2000 hours of Factorio in the last 5 years. I know that to a lot of people, those are rookie numbers, but it’s been long enough for me to get a good idea of how the game works and to work through a number of mod-packs. I started off with completely vanilla runs, restarting a few times when I realised I’d made some kind of hideous abomination, until I finally launched my first rocket. I then messed around with railworlds and various megabase attempts before starting my first big [Youtube](https://www.youtube.com/c/LaurencePlays) series back in July 2019 with an [AngelBobs plus Space Expansion](https://www.youtube.com/watch?v=bQvCHSRVcqg&list=PLALTzJaRcgW9qOavYQJ15M0pSoiVedG18) run. This took me slightly more than a year, but eventually I managed to finish the faster than light research and called that game done.
+
+My next modpack was [Space Exploration](https://www.youtube.com/watch?v=oOHkLsSk1wo&list=PLALTzJaRcgW9iPN4iYNTBlz03XpAvkIlD). I started this run pretty soon after finishing AngelBobs and am now about a year and a half in, and I think I’m mostly done, but it will probably take a good few months longer to actually finish it. Finally, in parallel to Space Exploration, I played through [Industrial Revolution 2](https://www.youtube.com/watch?v=LWrAZkapDTU&list=PLALTzJaRcgW8HveofZ0iPpg0u3g6llmNc) with some friends during the second half of 2021. 
+
+So, now I’ve given you a quick history of my Factorio experience, I can start on the main topic of this article – comparing the different mod packs, what they give you and how they differ in feel. I have also made a video version of this article, which you can watch [here](link to follow).
+
+### Vanilla
+
+The original game is, of course, the baseline. It’s where we (probably!) all started and where everything else has grown from. Most of the recipes are quite simple, in that they generally only have a single output – no byproducts to worry about – and there’s usually only one way to make anything you might need. The game starts off very simple with red science packs taking a mere 4 steps – smelting the iron, smelting the copper, making gears and making the pack – but gradually building up. Blue science has traditionally been the great filter – the point at which a number of players find the game has become difficult, due to having to deal with oil, and then it gets significantly harder again with yellow science, largely due to the complexities of making blue circuits. Or rather, the challenge of making green circuits fast enough to keep up with the requirements for blue!
```

Comment:
Here we can say that Vanilla started with inspiration from Modded Minecraft :P
https://alt-f4.blog/ALTF4-34/